src/test/resources/services.conf

```diff
@@ -32,13 +32,13 @@
 services {
     auth-login-api {
       protocol = http
-#     host = **place holder **
+      host = www.staging.tax.service.gov.uk
```

Comment:
This url is wrong. Should be auth-login-api.protected.mdtp

Reply:
Replaced with given url.